src/components/curation/PublishSuccessAlert.vue

```diff
@@ -30,7 +30,7 @@ export default {
     <hr />
     <div class="d-flex justify-content-between">
       <router-link :to="`/lgd/${stableId}`" class="btn btn-primary me-3">
-        View New G2P Record
+        View New G2P Record {{ stableId }}
```

Comment:
Suggesting we use `View G2P Record {{ stableId }}` instead

Reply:
taken care